src/inout.c

```diff
@@ -118,6 +118,71 @@ int H2T_loadNpy(MMG5_pMesh mmgMesh, int** tabhex, char* filename) {
     }
   }
 
+  /* Edges */
+  ref = 0;
+  pos = 0;
+  for (i=0;i<t[0]-1; ++i) {
+    int p0, p1;
```

Comment:
Is it possible to rename this variables `np0` and `np1`? 

It is a detail and this is probably not respected everywhere in the code but I think that it is a good habit to use some naming convention in variables name: `p<i>` or `ppt` for point structure or (`MMG5_Point`) pointers toward point structure (`MMG5_pPoint`), `ip<i>` for local point index and `np<i>` for global point index (`int`).

It allows an easier understanding for complex or large functions (without having to search or remember what the variable is used for.

Reply:
This modification been done as requested.